src/Middleware/WebSockets/src/WebSocketOptions.cs

```diff
@@ -23,6 +25,32 @@ public WebSocketOptions()
     /// </summary>
```

Comment:
Should we update this to match `WebSocketAcceptContext.KeepAliveInterval` or vice-versa?